.travis.yml

```diff
@@ -53,7 +53,7 @@ matrix:
         - sphinx-build -W -b html -d _build/doctrees source _build/html
         - cd ..
         # Run Python linting, ignore dict vs {} (C408), others are defaults
-        - flake8 --exclude=python/ray/core/generated/,doc/source/conf.py,python/ray/cloudpickle/ --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504,W605
+        - flake8 --inline-quotes '"' --exclude=python/ray/core/generated/,doc/source/conf.py,python/ray/cloudpickle/ --ignore=C408,E121,E123,E126,E226,E24,E704,W503,W504,W605
```

Comment:
Should this be inside the format.sh script?

Reply:
Yes, good call. @justinwyang please add this to `format.sh` as well.

Reply:
@ericl The format.sh file has two calls to flake8, is this to be applied to both calls or just the first one?

Reply:
Probably both, unless we want to exclude some files for some reason.

Reply:
Shall we add a project-wide flake8 config file? then we don't need to worry about  inconsistent configs in different scripts.